include/linux/minmax.h

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef __BACKPORT_MINMAX_H
+#define __BACKPORT_MINMAX_H
+
+#include <linux/version.h>
+
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 10, 0)
```

Comment:
The linux/minmax.h file is included i CentOS8.5 and the following RockyLinux 8.6 and 8.7. So the criteria should be:

#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 10, 0) && RHEL_RELEASE_CODE <= 0x804